### PR TITLE
Added scripted solution to ssl page.

### PR DESCRIPTION
**Pull request by emartinez-usgs**

### CODE REVIEW

**Comment by isuftin** [comment]
I also have this process in a Chef cookbook along with the process to update Java with the SSL certs. Probably not to be used with this PR but good to know anyway: https://github.com/USGS-CIDA/chef-cookbook-doi-ssl-filtering

File: WorkingWithinSSLIntercept.md [hunk 1]
```diff
@@ -15,7 +15,27 @@ openssl -in CustomCA.der -inform DER -out CustomCA.crt -outform PEM
 
 ## Linux:
 
-On linux systems the `.crt` extension matters.
+On linux systems the `.crt` extension matters. The following script may be
+of some use:
```
Comment (cschroed-usgs) [inline WorkingWithinSSLIntercept.md]: Thanks for submitting this script! Could you please clarify in the prose which Linux distros this has been tested on?

**Comment by cschroed-usgs** [comment]
Thanks for shouting out that chef cookbook @isuftin! @emartinez-usgs do you think it would be pertinent to include the url to the chef cookbook in this best practices repo?

**Comment by emartinez-usgs** [comment]
@cschroed-usgs Yes I was looking at that just now. I'll add a link for now. Eventually we need to organize this entire repo better (IMO) but who has the time?

**Pull request merged**